test/mediatype.test.js

```diff
@@ -62,7 +62,7 @@ describe("body with content-type application/xml", function () {
     describe("with valid content", function () {
         let uploadResponse
         const filename = 'valid.xml'
-        const contents = Buffer.from('<root/>')
+        const contents = Buffer.from('<root><p>valid xml</p></root>')
```

Comment:
Looking at the failed test we have to assume that the given XML is not stored verbatim into the database.
The assertion in Line 83 below needs to be changed to either
- strip all whitespace from the data in the response
- use a less strict comparison